lib/videojs5-hlsjs-source-handler.js

```diff
@@ -52,6 +52,22 @@ var attachVideojsStreamrootProvider = function (window, videojs, Hls) {
 
         function load(source) {
             _hls.loadSource(source.src);
+
+            if (!_hls.config.autoStartLoad) {
+                var checkStartLoad = function(e) {
+                    e.target.removeEventListener(e.type, checkStartLoad);
+                    if (typeof _hls.streamController.levels !== 'undefined') {
+                        _hls.startLoad(_hls.config.startPosition);
+                    } else {
+                        var startLoad = function() {
+                            _hls.off(Hls.Events.MANIFEST_PARSED, startLoad);
+                            _hls.startLoad(_hls.config.startPosition);
+                        };
+                        _hls.on(Hls.Events.MANIFEST_PARSED, startLoad);
+                    }
+                };
+                _video.addEventListener('play', checkStartLoad);
```

Comment:
Adding it to _video was quick and easy for me since I am not very familiar with Video.js internals :)

Can you give me an example on how to add a play listener using a tech event?